src/Apps/ArtAdvisor/07-Curated-Discovery/App.tsx

```diff
@@ -1,31 +1,41 @@
-import { Box, Button, Join, Spacer, Text } from "@artsy/palette"
 import { FC, useReducer } from "react"
-import { Interests } from "./Components/Interests"
-import { Goals } from "./Components/Goals"
-import { Budget } from "Apps/ArtAdvisor/07-Curated-Discovery/Components/Budget"
+import { Form } from "./Components/Form/Form"
+import { Result } from "./Components/Result/Result"
+
+type Step = "form" | "result"
+
+export type BudgetIntent = {
+  amount: number
+}
```

Comment:
This will grow to reflect something like [this logic](https://www.notion.so/artsy/Curated-Discovery-5e43238042014fae853222582a35acd9?pvs=4#459e705629044ccbbbe56d589ad12cf3) in a future PR